### PR TITLE
CB-421: Show user ratings on the profile page

### PR DESCRIPTION
**Pull request by anshg1214**
[CB-421](https://tickets.metabrainz.org/projects/CB/issues/CB-421) Added User ratings on the profile page.

### CODE REVIEW

File: critiquebrainz/frontend/templates/user/base.html [hunk 1]
```diff
@@ -30,3 +30,8 @@ <h2 class="pull-left"><img class="avatar" src="{{ user.avatar }}&s=32" /> {{ use
     {% block profile_content %}{% endblock %}
   </div>
 {% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script src="{{ get_static_path('rating.js') }}"></script>
```
Comment (amCap1712) [inline critiquebrainz/frontend/templates/user/base.html:36]: This script is not used in this file and user/info.html inherits from this template and does not use this script either. So I suggest to move this block to reviews.html which is the file actually needing the script.

File: critiquebrainz/frontend/messages.pot [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Translations template for PROJECT.
-# Copyright (C) 2021 ORGANIZATION
+# Copyright (C) 2022 ORGANIZATION
```
Comment (amCap1712) [inline critiquebrainz/frontend/messages.pot:2]: I don't think there are translation updates to apply as only metadata appears to change. So I suggest to revert changes to the file.

**Review by amCap1712 — CHANGES_REQUESTED** [review]
Hi! Thanks for working on this. The changes look good to me. Once the feedback is addressed, I'll merge the PR.

File: critiquebrainz/frontend/messages.pot (hunk 2 above)
Comment (anshg1214) [reply critiquebrainz/frontend/messages.pot:2]: There was a new column `ratings` added, so I added it for translation. Also, due to the addition of code, line numbers had changed.  So, I updated the file.

File: critiquebrainz/frontend/templates/user/base.html (hunk 1 above)
Comment (anshg1214) [reply critiquebrainz/frontend/templates/user/base.html:36]: Sure, I'll do that.

File: critiquebrainz/frontend/messages.pot (hunk 2 above)
Comment (amCap1712) [reply critiquebrainz/frontend/messages.pot:2]: Ah! This change is fine then.

**Review by amCap1712 — APPROVED** [review]
Thanks!

**Pull request merged**